inc/abstracttarget.class.php

```diff
@@ -1551,7 +1551,7 @@ protected function showValidationSettings($rand) {
       echo '<td width="25%">';
 
       // COMMONITIL_VALIDATION_RULE_SPECIFIC_USER_OR_GROUP
-      $display = $this->fields['commonitil_validation_answer_user'] == self::COMMONITIL_VALIDATION_RULE_SPECIFIC_USER_OR_GROUP ? "" : "display: none";
+      $display = $this->fields['commonitil_validation_question'] == self::COMMONITIL_VALIDATION_RULE_SPECIFIC_USER_OR_GROUP ? "" : "display: none";
```

Comment:
Seems like it should be `commonitil_validation_rule`.
```suggestion
      $display = $this->fields['commonitil_validation_rule'] == self::COMMONITIL_VALIDATION_RULE_SPECIFIC_USER_OR_GROUP ? "" : "display: none";
```

Also can you fix this one too ?
![image](https://user-images.githubusercontent.com/42734840/151135336-a90960de-59f3-41a5-900d-4157d46ab585.png)